cli/internal/runcache/runcache.go

```diff
@@ -225,8 +227,10 @@ func (tc TaskCache) OutputWriter() (io.WriteCloser, error) {
 	if err != nil {
 		return nil, err
 	}
+	pref := tc.colorCache.PrefixColor(tc.pt.PackageName)
```

Comment:
Actual strong opinion: `pref` is wired to "preferences" In my brain, can we maybe call this `prefixColor`?

Reply:
`pref` is a function here (given a string, it colors it) - so I might tweak this slightly to be `colorPrefixer`